recipe/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
     - libgsasl
     - libntlm
     - libuuid
-    - libxml2
+    - libxml2  {{ libxml2 }}
```

Comment:
please replace libxml2 by 2.10 here. no need to add it to a private cbc

Reply:
Done